runtime/vm/CRIUHelpers.cpp

```diff
@@ -400,6 +417,23 @@ runInternalJVMRestoreHooks(J9VMThread *currentThread)
 		hookRecord = (J9InternalHookRecord*)pool_nextDo(&walkState);
 	}
 
+	if ((NULL != classIterationRestoreHookRecords) && (pool_numElements(classIterationRestoreHookRecords)) > 0) {
+		J9ClassWalkState j9ClassWalkState = {0};
+		J9Class *clazz = vm->internalVMFunctions->allClassesStartDo(&j9ClassWalkState, vm, NULL);
+		while (NULL != clazz) {
+			J9InternalClassIterationRestoreHookRecord *hookRecord = (J9InternalClassIterationRestoreHookRecord*)pool_startDo(classIterationRestoreHookRecords, &walkState);
+			while (NULL != hookRecord) {
+				if (FALSE == hookRecord->hookFunc(currentThread, clazz)) {
```

Comment:
can just assign return val to result

Reply:
Assigned the return val to `result`, also updated L412 as well.
@tajila this is ready for another look.